Librarian/Series.js

```diff
@@ -73,7 +69,7 @@ Series.prototype._inspect = (seriesItem, done)=> {
     if (err) {
       return done(err);
     }
-    var update = {
+    const update = {
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).